### PR TITLE
Design revisions

### PR DESCRIPTION
**Pull request by alecmarcus**
# What this does

- New typography scale
- Slightly revised home page design
- Adds missing styles to markdown articles:
  - tables
  - lists
  - links
  - blockquotes
- Fixes some style bugs
  - `appearance: none` in safari
  - incorrect colors in `pre`s inside `li`s
- adds a two-up-code block class
- use a light syntax highlight theme
- use a mixin for page content columns
- tweak grayscale
- edits the HTML in JS of `SearchBar.js` to use `templates`, allowing for more visual parity between the source markup in `home/index/index.njk` and the replicated markup

# Blockers
- [x] Merge #39

### CODE REVIEW

File: .eleventy.js [hunk 1]
```diff
@@ -13,9 +13,6 @@ const {
   htmlDateString,
   readableDate,
   split,
-  // includesSolution,
-  // includesTeam,
-  // includesTechnology,
```
Comment (bchiang7) [inline .eleventy.js:18]: can we get rid of these?

**Review by bchiang7 — APPROVED** [review]
LGTM!

File: src/series/learn-git.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+layout: layouts/series/index.njk
+title: Learn Git
```
Comment (stephalee) [inline src/series/learn-git.md:3]: Just a thought — I wonder if we called these series' something like `Git Fundamentals` or `Git Basics`. Take or leave the idea!

**Review by stephalee — APPROVED** [review]
Added one comment but this is overall this is awesome!!! 🙌 Such a huge improvement in so many ways

**Pull request merged**